spyder/plugins/application/widgets/status.py

```diff
@@ -116,11 +116,20 @@ def _on_click(self):
     # ---- Public API
     # -------------------------------------------------------------------------
     def show_appeal(self):
-        if self._appeal_dialog is None:
-            self._appeal_dialog = InAppAppealDialog(self)
-
-        if not self._appeal_dialog.isVisible():
-            self._appeal_dialog.show()
+        from qtpy import QtModuleNotInstalledError
```

Comment:
Please move this import to the top with the other imports. I don't think there's a need to keep it here.